draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -740,15 +720,15 @@ If any step fails, the Presentation is not valid and processing MUST be aborted.
 
 Otherwise, the processed SD-JWT payload can be passed to the application to be used for the intended purpose.
```

Comment:
```suggestion
Otherwise, the processed Issuer-signed JWT payload can be passed to the application to be used for the intended purpose.
```

Reply:
I believe "SD-JWT payload" is accurate (enough) and okay here.

Reply:
ok..